packages/sanity/test/setup/environment.ts

```diff
@@ -22,8 +22,8 @@ export {}
 // get rid of context warning
 const warn = console.warn
 const error = console.error
-window.console = {
-  ...window.console,
+;(globalThis as any).console = {
```

Comment:
not strictly related to the changes here, but needed in order to run tests in node